.github/workflows/ci.yml

```diff
@@ -6,38 +6,16 @@ on:
       - master
       - 'feature/**'
       - 'v**'
+      - 'test/v**'
   pull_request:
     branches:
       - master
       - 'feature/**'
       - 'v**'
+      - 'test/v**'
 
 jobs:
-  paths-filter:
```

Comment:
What is the reasoning behind removing the paths filter?

Reply:
no reason, it must have inadvertantly got removed 🙃

Reply:
Sounds good, no worries!